supertinycron/examples/cron_example/CMakeLists.txt

```
@@ -0,0 +1,8 @@
+# For more information about build system see
+# https://docs.espressif.com/projects/esp-idf/en/latest/api-guides/build-system.html
+# The following five lines of boilerplate have to be in your project's
+# CMakeLists in this exact order for cmake to work correctly
+cmake_minimum_required(VERSION 3.16)
+
+include($ENV{IDF_PATH}/tools/cmake/project.cmake)
```

```suggestion
set(COMPONENTS main)
include($ENV{IDF_PATH}/tools/cmake/project.cmake)
```

just to reduce build time by default

---

Done